### PR TITLE
test of an idea

### PR DESCRIPTION
**Pull request by i4ki**
This implements the idea proposed in #1 

I've changed the coverage to make it drop-in replacement in nash project.

Feel free to say if this sucks and just redirect to /dev/null 
=)

### CODE REVIEW

**Comment by i4ki** [comment]
Example of use:

```
λ> import gootstrap/cover
λ> cover("github.com/c0defellas/enzo")
=== RUN   TestCatInOut
=== RUN   TestCatInOut/#00
=== RUN   TestCatInOut/test
=== RUN   TestCatInOut/AAAAAAAAAAAAAAAAAAAAAAAAAA
=== RUN   TestCatInOut/1_2_3_4_5_6_7_8_9_10_
--- PASS: TestCatInOut (0.00s)
    --- PASS: TestCatInOut/#00 (0.00s)
    --- PASS: TestCatInOut/test (0.00s)
    --- PASS: TestCatInOut/AAAAAAAAAAAAAAAAAAAAAAAAAA (0.00s)
    --- PASS: TestCatInOut/1_2_3_4_5_6_7_8_9_10_ (0.00s)
=== RUN   TestHandlesMultipleFiles
--- PASS: TestHandlesMultipleFiles (0.00s)
=== RUN   TestHandleOneFile
--- PASS: TestHandleOneFile (0.00s)
=== RUN   TestHandleFileNotFound
--- PASS: TestHandleFileNotFound (0.00s)
=== RUN   TestHandleReadError
--- PASS: TestHandleReadError (0.00s)
=== RUN   TestHandleWriteError
--- PASS: TestHandleWriteError (0.00s)
PASS
coverage: 63.3% of statements
ok  	github.com/c0defellas/enzo/cmd/cat	1.024s
=== RUN   TestEcho
--- PASS: TestEcho (0.00s)
=== RUN   TestParseArg
--- PASS: TestParseArg (0.00s)
PASS
coverage: 88.2% of statements
ok  	github.com/c0defellas/enzo/cmd/echo	1.012s
=== RUN   TestKillUnsafe
=== RUN   TestKillUnsafe/kill_12821
=== RUN   TestKillUnsafe/kill_12822
=== RUN   TestKillUnsafe/kill_12823
=== RUN   TestKillUnsafe/kill_12824
=== RUN   TestKillUnsafe/kill_12825
=== RUN   TestKillUnsafe/kill_12826
=== RUN   TestKillUnsafe/kill_12827
=== RUN   TestKillUnsafe/kill_12828
=== RUN   TestKillUnsafe/kill_12829
=== RUN   TestKillUnsafe/kill_12830
=== RUN   TestKillUnsafe/kill_12831_12832
=== RUN   TestKillUnsafe/kill_12833_12834_12835
=== RUN   TestKillUnsafe/kill_12836_12837_12838_12839_12840_12841_12842
=== RUN   TestKillUnsafe/kill_12843_12844_12845_12846_12847
=== RUN   TestKillUnsafe/kill_12848_12849_12850_12851_12852_12853_12854_12855_12856
--- PASS: TestKillUnsafe (0.04s)
    --- PASS: TestKillUnsafe/kill_12821 (0.00s)
    --- PASS: TestKillUnsafe/kill_12822 (0.00s)
    --- PASS: TestKillUnsafe/kill_12823 (0.00s)
    --- PASS: TestKillUnsafe/kill_12824 (0.00s)
    --- PASS: TestKillUnsafe/kill_12825 (0.00s)
    --- PASS: TestKillUnsafe/kill_12826 (0.00s)
    --- PASS: TestKillUnsafe/kill_12827 (0.00s)
    --- PASS: TestKillUnsafe/kill_12828 (0.00s)
    --- PASS: TestKillUnsafe/kill_12829 (0.00s)
    --- PASS: TestKillUnsafe/kill_12830 (0.00s)
    --- PASS: TestKillUnsafe/kill_12831_12832 (0.00s)
    --- PASS: TestKillUnsafe/kill_12833_12834_12835 (0.00s)
    --- PASS: TestKillUnsafe/kill_12836_12837_12838_12839_12840_12841_12842 (0.00s)
    --- PASS: TestKillUnsafe/kill_12843_12844_12845_12846_12847 (0.00s)
    --- PASS: TestKillUnsafe/kill_12848_12849_12850_12851_12852_12853_12854_12855_12856 (0.00s)
PASS
coverage: 13.9% of statements
ok  	github.com/c0defellas/enzo/cmd/kill	1.056s
=== RUN   TestListCurrentDir
--- PASS: TestListCurrentDir (0.00s)
=== RUN   TestListCurrentDirAsAList
--- PASS: TestListCurrentDirAsAList (0.00s)
=== RUN   TestHumanizeBytes
--- PASS: TestHumanizeBytes (0.00s)
PASS
coverage: 48.0% of statements
ok  	github.com/c0defellas/enzo/cmd/ls	1.014s
github.com/c0defellas/enzo/cmd/cat/cat.go:11:		fatal		0.0%
github.com/c0defellas/enzo/cmd/cat/cat.go:17:		cat		100.0%
github.com/c0defellas/enzo/cmd/cat/cat.go:42:		runcat		88.9%
github.com/c0defellas/enzo/cmd/cat/cat.go:58:		main		0.0%
github.com/c0defellas/enzo/cmd/echo/echo.go:8:		echo		100.0%
github.com/c0defellas/enzo/cmd/echo/echo.go:24:		parsearg	100.0%
github.com/c0defellas/enzo/cmd/echo/echo.go:41:		main		0.0%
github.com/c0defellas/enzo/cmd/kill/kill_unix.go:9:	kill		41.7%
github.com/c0defellas/enzo/cmd/kill/main.go:10:		usage		0.0%
github.com/c0defellas/enzo/cmd/kill/main.go:17:		sliceatoi	0.0%
github.com/c0defellas/enzo/cmd/kill/main.go:33:		parseargs	0.0%
github.com/c0defellas/enzo/cmd/kill/main.go:48:		main		0.0%
github.com/c0defellas/enzo/cmd/ls/ls.go:21:		fatal		0.0%
github.com/c0defellas/enzo/cmd/ls/ls.go:26:		humanizeSize	100.0%
github.com/c0defellas/enzo/cmd/ls/ls.go:39:		formatFileName	100.0%
github.com/c0defellas/enzo/cmd/ls/ls.go:46:		printFileList	100.0%
github.com/c0defellas/enzo/cmd/ls/ls.go:54:		printFileNames	100.0%
github.com/c0defellas/enzo/cmd/ls/ls.go:58:		ls		100.0%
github.com/c0defellas/enzo/cmd/ls/ls.go:64:		main		0.0%
total:							(statements)	47.2%
```

File: coverage.sh [hunk 1]
```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env nash
```
Comment (katcipis) [inline coverage.sh:1]: Why you are removing the coverage ? :-(

**Review by katcipis — CHANGES_REQUESTED** [review]

File: coverage.sh (hunk 1 above)
Comment (katcipis) [reply coverage.sh:1]: Ow I see now that you said : I've changed the coverage to make it drop-in replacement in nash project."

But I have no idea what you mean with that :-(

Comment (i4ki) [reply coverage.sh:1]: kkk I renamed the script to cover.sh because I'm lazy kkk If you dislike we can use coverage.sh :-)

**Comment by i4ki** [comment]
@katcipis Apart from the script file name, what do you think of script changes I've made? 
Nonsense?

**Comment by katcipis** [comment]
LGTM

**Pull request merged**